### DISCUSSION TITLE
How to switch files like vscode ?

### DISCUSSION THREAD

**Discussion by AaronConlon** [ecosse3/nvim#112] [Q&A] [answered]
So thanks.I had read readme doc,but I donot find it.

**Comment by AaronConlon** [comment]
More question:
- how to auto save and format code ?
- how to close current file tab?

**Reply by jvegaf** [reply]
you can close tabs with Q (shift-q)

**Reply by jvegaf** [reply]
> * how to auto save and format code ?

First **\<SPACE-C\>** to open LSP settings, and after press **\<T\>** for activate **format on save**
 
you can find that option [here](https://github.com/ecosse3/nvim/blob/master/lua/plugins/which-key.lua#L146)

**Comment by jvegaf** [comment, marked as answer]
> So thanks.I had read readme doc,but I donot find it.

Although there is no documentation, you can find the shortcuts in the [keymaps file](https://github.com/ecosse3/nvim/blob/master/lua/keymappings.lua)

**Reply by edheltzel** [reply]
updated link to [keymaps](https://github.com/ecosse3/nvim/blob/master/lua/config/keymappings.lua)
